### DISCUSSION TITLE
Cannot insert row in mssql database table with active triggers

### DISCUSSION THREAD

**Discussion by flo-hame** [aarondl/sqlboiler#992] [Q&A]
Hello,
currently it is not possible for me, creating a new row in a mssql database table with an active trigger.
When using the article.Insert() function, I am receiving the error:

unable to insert into Article: mssql: The target table 'dbo.Article' of the DML statement cannot have any enabled triggers if the statement contains an OUTPUT clause without INTO clause.

For breaking it down, this happens because the generated query from sqlboiler automatically adds an OUTPUT - query to the insert query. But this is not allowed when having active triggers.
Is there a way to prevent adding this OUTPUT - query, or do I have to write raw Queries ?

**Comment by aarondl** [comment]
That's quite the odd restriction (and one I wasn't aware of). We use the OUTPUT clause to bind things back into your struct, otherwise they end up out of sync.

What is the fix using `INTO`?

**Reply by stephenafamo** [reply]
Apparently, the best thing to do is to run an extra select query after to retrieve the result according to this:

https://stackoverflow.com/a/13198916/5164997
